index.bs

```diff
@@ -1086,13 +1086,12 @@ run the following steps:
     1. If the [=byte sequence=] [=byte sequence/length=] of |encodedDomain| is greater than the [=cookie/maximum attribute value size=], then return failure.
     1. [=list/Append=] \``Domain`\`/|encodedDomain| to |attributes|.
 1. If |expires| is given, then [=list/append=] \``Expires`\`/|expires| ([=date serialized=]) to |attributes|.
-1. If |path| is not null:
-    1. If |path| does not start with U+002F (/), then return failure.
-    1. If |path| is not U+002F (/), and |name|, [=byte-lowercased=], [=byte sequence/starts with=] \``__host-`\`, then return failure.
-    1. Let |encodedPath| be the result of [=UTF-8 encode|UTF-8 encoding=] |path|.
-    1. If the [=byte sequence=] [=byte sequence/length=] of |encodedPath| is greater than the [=cookie/maximum attribute value size=], then return failure.
-    1. [=list/Append=] \``Path`\`/|encodedPath| to |attributes|.
-1. Otherwise, [=list/append=] \``Path`\`/ U+002F (/) to |attributes|.
+1. If |path| is the empty string, then set |path| to the [=/URL path serializer|URL path serialization=] of |url|.
```

Comment:
Since |path| is an optional parameter, I still think we need to handle the case when it is not provided, i.e. "path is null," and have that default to `"/"` like we have been.

Reply:
Are you talking about the case where `set()` is given _options_? IDL handles it for us there. Also, if IDL didn't handle it that would mean _path_ didn't exist in _options_ and we'd hit a different kind of problem earlier on. It can't be null here.